docs/API/LorisRESTAPI_v0.0.3.md

```diff
@@ -238,6 +238,31 @@ will return a JSON object of the form:
 
 where 123456, 342332, etc are the candidates that exist for this project.
 
+#### 2.1.5 Single project electrophysiology recordings
+```
+GET /projects/$ProjectName/recordings/
```

Comment:
`GET /projects/$ProjectName/recordings/`
@cmadjar is the trailing slash needed?

Reply:
looks like it is not needed so I removed it from the documentation. Thanks @maltheism !